app/Events/HoldAssignedEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Events;
+
+use App\Models\Hold\AssignedHold;
+use Illuminate\Broadcasting\Channel;
+use Illuminate\Broadcasting\PrivateChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class HoldAssignedEvent implements ShouldBroadcast
+{
+    const CHANNEL = 'hold-assignments';
```

Comment:
Would this be better off as a config variable given it is defined in multiple files?

Reply:
I think this is the best place, for now - this isn't something we're going to change on a whim, it's going to have to be very much coordinated as it's written into the C++ to receive it there :)